apps/css-workshop/src/pages/toast.astro

```diff
@@ -47,27 +48,27 @@ import IconButton_ from '../components/IconButton.astro';
     <div class='placement-control'>
       <h3>Toast placement:</h3>
       <label class='placement-choice' id='top-start-option'>
-        <input type='radio' name='radio-demo-6' value='top-start' />
+        <Radio_ name='radio-demo-6' value='top-start' />
```

Comment:
These radios do not rely on our CSS (there is no `"iui-radio"` class) so they do not need to be componentized. You can revert the changes in this file, and simplify `Radio.astro` to remove the `basic` handling.

Reply:
I have made the change.

Reply:
Can you review the current state of your own PR to make sure there are no leftovers in `toast.astro`?

Also, the `inputProps` in `Radio.astro` does not need to be a variable anymore; the props can be used directly on the `<input>`

Reply:
I have checked `toast` page and other pages and cleaned up the `Radio` component.